dvc/repo/imp.py

```diff
@@ -4,6 +4,6 @@
 def imp(self, url, path, out=None, rev=None):
     erepo = {ExternalRepo.PARAM_URL: url}
     if rev is not None:
-        erepo[ExternalRepo.PARAM_VERSION] = rev
+        erepo[ExternalRepo.PARAM_REV] = rev
```

Comment:
what is the syntax to specify a specific branch, is it part of the URL? does `rev` now accept only commit hash syntax?

Reply:
can I specify a git tag? (and lock it in case it's moving)

Reply:
Yes, you can specify any valid git revision (including sha, tag and branch). The syntax is the same `dvc import/get URL --rev MYREV`.